assets/RNAseq_report.Rmd

```diff
@@ -1270,19 +1264,20 @@ Here, volcano plots for all considered contrasts are shown. Hover over the dots
     log2FoldChange_min <- min(DE_all$log2FoldChange)
     log2FoldChange_max <- max(DE_all$log2FoldChange)
     pg <- ggplot(DE_all, aes(x=log2FoldChange, y=logpval, text=paste("Gene: ", gene_name, "<br>", "Log2FC: ", formatC(log2FoldChange, digits=2)))) +
-            geom_point(alpha=0.5, aes(color=ifelse(abs(DE_all$log2FoldChange) > params$log_FC_threshold &  DE_all$logpval > -log10(pval_threshold), "Differentially expressed genes", "Non-differentially expressed genes"))) +
+            #this goes after alpha=0.5:
+            geom_point(alpha=0.5, aes(color=ifelse(abs(DE_all$log2FoldChange) > params$logFC_threshold &  DE_all$logpval > -log10(params$pval_threshold), "Differentially expressed genes", "Non-differentially expressed genes"))) +
             geom_hline(yintercept = 16, linetype= "dashed", size = 0.2, color = "grey") +
-            geom_hline(yintercept = -log10(pval_threshold), size = 0.2, color = "grey") +
-            geom_vline(xintercept = params$log_FC_threshold, size = 0.2, color = "grey") +
-            geom_vline(xintercept = -params$log_FC_threshold, size = 0.2, color = "grey") +
+            geom_hline(yintercept = -log10(params$pval_threshold), size = 0.2, color = "grey") +
+            geom_vline(xintercept = -params$logFC_threshold, size = 0.2, color = "grey") +
+            geom_vline(xintercept = params$logFC_threshold, size = 0.2, color = "grey") +
             scale_color_manual(name = "", values=c("red", "black")) +
             scale_y_continuous(limits = c(1, 18)) +
             scale_x_continuous(breaks = seq(-200, 200, 1)) +
             facet_wrap(~contrast, ncol=1, scales = "free_x", shrink = FALSE) +
             ylab("-log 10 padj") + xlab("Log2 Fold Change")
     pg <- ggplotly(pg, tooltip = "text")
-    pg[['x']][['layout']][['annotations']][[2]][['x']] <- -0.05
-    pg[['x']][['layout']][['annotations']][[1]][['y']] <- -0.05
+    pg[['x']][['layout']][['annotations']][[2]][['x']] <- 5 #TODO: what do these two lines do?
```

Comment:
Make sure you know what will happen if you remove them

Reply:
I honestly still have no clue what they do...I set the second line to 200 and the plot just did not change at all...I am resetting them to -0.05, maybe sometime in the future I can find out their function